composer.json

```diff
@@ -50,7 +50,7 @@
     "verbb/element-index-defaults": "^4.0",
     "verbb/field-manager": "^4.0",
     "verbb/image-resizer": "^4.0",
-    "verbb/navigation": "^3.0",
+    "verbb/navigation": "3.0.8",
```

Comment:
We don’t lock plugin versions even for paid plugins in the boilerplate because we want the latest version of the paid plugin to be used when the boilerplate is cloned to spin off a new project. So let us retain this as `^3.0`.